src/xrpld/app/tx/detail/Transactor.cpp

```diff
@@ -760,6 +761,19 @@ removeExpiredNFTokenOffers(
     }
 }
 
+static void
+removeExpiredCredentials(
+    ApplyView& view,
+    std::vector<uint256> const& creds,
+    beast::Journal viewJ)
+{
+    for (auto const& index : creds)
+    {
+        if (auto const sle = view.peek(keylet::credential(index)))
+            credentials::deleteSLE(view, sle, viewJ);
```

Comment:
This looks like it doesn't handle the `OwnerCount` changes.

Reply:
Please check `deleteSLE` code, it does